lib/themes/dosomething/paraneue_dosomething/js/utilities/LazyLoader.js

```diff
@@ -0,0 +1,50 @@
+import Layzr from 'layzr.js';
+
+// Application lazy loader instance.
+let loader = null;
+
+/**
+ * Helper function to check for new [data-src] elements that were added
+ * after DOMContentLoaded and lazy-load them.
+ */
+export function update() {
+  if (loader === null) {
+    throw new Error('Lazy loader has not been initialized.');
+  }
+
+  loader.update().check();
+}
+
+/**
+ * Initialize a new lazy loader.
+ *
+ * @returns Layzr
+ */
+export function init() {
+  if (loader !== null) {
+    return loader;
+  }
+
+  loader = new Layzr({
+    normal: 'data-src',
+    retina: 'data-src-retina',
+  });
+
+  loader.on('src:before', image => {
+    // On load, set the images to be transparent so we can fade them in.
+    image.style.transition = 'opacity 0.5s ease-in';
+    image.style.opacity = 0;
```

Comment:
Any chance we would want to use/remove classes here instead of adding properties to `style` object? Would it make it easier to modify the effect from CSS rather than JS? Just a thought.